backend-project/small_eod/letters/serializers.py

```diff
@@ -2,23 +2,38 @@
 from .models import Letter, Description
 from ..generic.serializers import UserLogModelSerializer
 
+from small_eod.files.serializers import FileSerializer
+
 
 class LetterSerializer(UserLogModelSerializer):
+    attachment = FileSerializer(many=True, read_only=True)
+
+    createdOn = serializers.CharField(source='created_on')
```

Comment:
Nie potrzebujesz tego, to powinno być generyczne. Poza tym zapisa python_case & camelCase rozwiązuje nam już Swagger.

Reply:
W takim razie usunę.